### PR TITLE
fix: update flow

### PR DESCRIPTION
**Pull request by jjmachan**
#### Description

- changed `operator remove` command to `operator uninstall` to make it more intuitive to the opposite command `operator install`
- Fixed the update workflow. Now `operator update <operator-name>` will update the operator to the latest version.
closes:

### CODE REVIEW

File: bentoctl/cli/operator_management.py [hunk 1]
```diff
@@ -103,13 +103,12 @@ def install(name=None, version=None):  # pylint: disable=unused-variable
         help="skip the prompt asking if you are sure.",
     )
     @click.argument("name", type=click.STRING)
-    def remove(name, skip_confirm):  # pylint: disable=unused-variable
+    def uninstall(name, skip_confirm):  # pylint: disable=unused-variable
```
Comment (ssheng) [inline bentoctl/cli/operator_management.py:106]: Let's leave `remove` as a deprecated command but not actually rename it.

File: bentoctl/cli/operator_management.py [hunk 2]
```diff
@@ -127,7 +126,7 @@ def remove(name, skip_confirm):  # pylint: disable=unused-variable
     @click.argument("name")
```
Comment (ssheng) [inline bentoctl/cli/operator_management.py:126]: Let's add an optional version argument. If not specified, the command will update to the latest.

File: bentoctl/operator/registry.py [hunk 3]
```diff
@@ -145,22 +154,44 @@ def install_operator(self, name, version=None):
...
+            logger.info(f"Operator is already on version {version}!")
+            return
+
+        repo_name = OFFICIAL_OPERATORS[name]
+        if isinstance(version, str):
+            updated_version_str = f"v{version.strip('v')}"
+        elif isinstance(version, Version):
+            updated_version_str = f"v{version}"
+        else:
+            raise BentoctlException("version is None")
```
Comment (ssheng) [inline bentoctl/operator/registry.py:175]: Update message to be more accurate.

File: bentoctl/operator/registry.py [hunk 4]
```diff
@@ -145,22 +154,44 @@ def install_operator(self, name, version=None):
...
+
             return name
-        except BentoctlException as e:
+        except Exception as e:
+            # undo all the changes
+            shutil.move(
+                os.path.join(tmp_operator_dir_path, operator.name), operator_path
+            )
+            self.operators_list[name]["version"] = f"v{operator.version}"
+            self._write_to_file()
```
Comment (ssheng) [inline bentoctl/operator/registry.py:194]: Can be move to finally.

File: bentoctl/operator/registry.py (hunk 3 above)
Comment (jjmachan) [reply bentoctl/operator/registry.py]: 👍️

**Review by ssheng — APPROVED** [review]

**Pull request merged**